.github/workflows/releases.yml

```diff
@@ -64,3 +64,15 @@ jobs:
         user: __token__
         password: ${{ secrets.PYPI_API_TOKEN }}
         verify_metadata: true
+
+  publish-rtd:
+    name: Publish docs in Readthedocs
+    runs-on: ubuntu-latest
+    needs: lint-unittests-docs
+    steps:
+    - name: Update readthedocs
+      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
No, it wouldn't because the `github.event_name` would be `release` (not `push`). I think any change should include an update of the `changelog.rst` file in the `docs/` folder.

Reply:
I agree, my point was what if I forgot to tag commit in PR and it will be merged. I will need to create another PR with what commit? Do you see my point?

Reply:
OK, so something like this would fix the problem you mention, wouldn't it?
`if: (github.event_name == 'release' || github.event_name == 'push') && startsWith(github.ref, 'refs/tags')`

Reply:
It's look like it will resolve it. I'm not saying your approach is wrong, but I see myself forgetting to tag commit in PR.
At the same time, we have the option to use the github release if we want.